### PR TITLE
Make the Delete button in the Edit profile page red

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/9515

---

Screen Shot 2020-07-13 at 19 39 35

### CODE REVIEW

File: src/amo/pages/UserProfileEdit/index.js [hunk 1]
```diff
@@ -807,7 +807,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     {submitButtonText}
                   </Button>
                   <Button
-                    buttonType="neutral"
+                    buttonType="alert"
```
Comment (eviljeff) [inline src/amo/pages/UserProfileEdit/index.js:810]: maybe we need some `<blink>` too 😆

**Comment by bobsilverberg** [comment]
I was just about to suggest this one to a contributor who was asking for issues, but now I see it's already done. Maybe we should leave some simple ones for contributors?

**Comment by eviljeff** [comment]
> I was just about to suggest this one to a contributor who was asking for issues, but now I see it's already done. Maybe we should leave some simple ones for contributors?

Once it was bumped to a p1 it wasn't very contributor appropriate any longer.

**Comment by bobsilverberg** [comment]
> > I was just about to suggest this one to a contributor who was asking for issues, but now I see it's already done. Maybe we should leave some simple ones for contributors?
> 
> Once it was bumped to a p1 it wasn't very contributor appropriate any longer.

Oops, I missed that. Makes sense. When it was first filed I noticed there was no priority set, so I assumed it was a P3.

**Pull request merged**